modules/data.remote/R/phenology_MODIS_extract.R

```diff
@@ -0,0 +1,115 @@
+##' Get MODIS phenology data by date and location  
+##' 
+##' @name phenology_MODIS_extract
+##' @title phenology_MODIS_extract
+##' @export
+##' @param settings PEcAn settings
+##' @param run_parallel optional method to download data parallely. Only works if more than 1 
+##' site is needed and there are >1 CPUs available.
+##' @param ncores number of cpus to use if run_parallel is set to TRUE. If you do not know the 
+##' number of CPU's available, enter NULL. 
+##' 
+##' The output file will be added to the folder defined in settings.
+##' 
+##' @examples
+##' \dontrun{
+##' settings <-  PEcAn.settings::read.settings("pecan.xml")
+##' test_modis_phenology <- PEcAn.data.remote::phenology_MODIS_extract(
+##'   settings = settings,
+##'   run_parallel = TRUE,
+##'   ncores = NULL)
+##' }
+##' @return NONE
```

Comment:
Does it really return nothing? From the example above I'd expect it to return a data frame of the requested data.

Reply:
A csv file will be generated and saved to a specific path defined in the setting. I can add the codes for returning a data frame if needed.